ext/spl/spl_directory.stub.php

```diff
@@ -292,5 +292,5 @@ public function __toString(): string {}
 
 class SplTempFileObject extends SplFileObject
 {
-    public function __construct(int $max_memory = 2 * 1024 * 1024) {}
+    public function __construct(int $maxMemory = 2 * 1024 * 1024) {}
```

Comment:
it could also be `$memoryLimit`, so that we use the same term as in case of `sodium_crypto_pwhash()`.

Reply:
I'm not sure on this one. My association with `$memoryLimit` is the PHP memory limit ^^ That is, fatal error if exceeded.

Reply:
That's fine, I'm not attached to this name at all :)